type_helpers.go

```diff
@@ -13,6 +13,36 @@ func ProjectAccess(v TeamProjectAccessType) *TeamProjectAccessType {
 	return &v
 }
 
+// ProjectSettingsPermission returns a pointer to the given team access project type.
+func ProjectSettingsPermission(v ProjectSettingsPermissionType) *ProjectSettingsPermissionType {
+	return &v
+}
+
+// ProjectTeamsPermission returns a pointer to the given team access project type.
+func ProjectTeamsPermission(v ProjectTeamsPermissionType) *ProjectTeamsPermissionType {
+	return &v
+}
+
+// WorkspaceRunsPermission returns a pointer to the given team access project type.
+func WorkspaceRunsPermission(v WorkspaceRunsPermissionType) *WorkspaceRunsPermissionType {
+	return &v
+}
+
+// WorkspaceSentinelMocksPermission returns a pointer to the given team access project type.
+func WorkspaceSentinelMocksPermission(v WorkspaceSentinelMocksPermissionType) *WorkspaceSentinelMocksPermissionType {
+	return &v
+}
+
+// WorkspaceStateVersionsPermission returns a pointer to the given team access project type.
+func WorkspaceStateVersionsPermission(v WorkspaceStateVersionsPermissionType) *WorkspaceStateVersionsPermissionType {
+	return &v
+}
+
+// WorkspaceStateVersionsPermission returns a pointer to the given team access project type.
+func WorkspaceVariablesPermission(v WorkspaceVariablesPermissionType) *WorkspaceVariablesPermissionType {
+	return &v
+}
```

Comment:
I still don't know how I felt about these pointer helpers in general. For me it was always easier/cleaner to use the address operator `&someStr` than `tfe.String(someStr)`. You can leave this if you feel it's helpful :)

Reply:
Jeff is out, so I can't remember what the issue quite was since my go is so new, but we def did also try the &someString to no avail and since it seemed an already established pattern we decided to use it.